lessons/measures-of-center/langs/en-us/pages/data-cycle-practice.adoc

```diff
@@ -14,7 +14,7 @@ Complete both of the Data Cycles shown here, which have questions defined to get
   #:show-filter? #f
   #:build-fn ""
   #:show-build? #f
-  #:expression "mean(filter(animals-table, is-cat)), \"age\")"
+  #:expression '(mean(filter(animals-table is-cat) "age"))
```

Comment:
@retabak minor nit for the future -- we always leave space after the function name in s-expressions/wescheme. Purely stylistic, so no need to change it here)